tox.ini

```diff
@@ -26,11 +26,11 @@ deps =
     django3.1: Django>=3.1,<3.2
     django3.2: Django>=3.2,<4.1
 
-    djangomaster: git+https://github.com/django/django.git@master#egg=Django
-    djangomaster: git+https://github.com/wagtail/django-modelcluster.git
+    djangomain: git+https://github.com/django/django.git@main#egg=Django
+    djangomain: git+https://github.com/wagtail/django-modelcluster.git
```

Comment:
suggestion: remove the django-modelcluster line. not used as such, also pollutes the `djangomain` env param

Reply:
Thanks @zerolab I have just split that to a new issue, will aim to work through any other suggestions tomorrow https://github.com/wagtail/wagtail-hallo/issues/12